en/docs/apis/create-and-publish-your-first-api.md

```diff
@@ -1,17 +1,13 @@
 # Create and Publish Your First API
 
-Choreo’s API management capabilities allow you to create, publish, and manage all aspects of an API and its lifecycle.
+Choreo API management capabilities allow you to create, publish, and manage all aspects of an API and its lifecycle.
 This quick start guide walks you through the steps to quickly create and publish a REST API using Choreo. 
  
 ## Step 1: Create an API from scratch
 Follow this procedure to create an API:
 
 1. Sign in to the Choreo Console at [https://console.choreo.dev/](https://console.choreo.dev/).
-2. Go to the **APIs** card and click **Get Started**. This takes you to the **Create API** page. 
-    
-    !!!note
-        If this is not the first API you are creating,  click **+ Create** to go to the **Create API** page.
-
+2. Go to the **APIs** card and click **Get Started**. This takes you to the **Create API** page.
```

Comment:
At present, the Dev environment has "Explore" instead of "Get Started" on the cards in the home page.